lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -617,7 +606,7 @@ sub _generate_jobs {
             splice @$ret, $i, 1;    # not wanted - delete
         }
     }
-    return $ret;
+    return {expand_result => $expand_result, settings_result => $ret};
```

Comment:
Don't use `expand_result` here as hash key. At the place where this is consumed the meaning of that name is not clear anymore. I'd prefer to use use `error` or `error_message`.